docker-compose.yml

```diff
@@ -75,6 +97,21 @@ services:
     depends_on:
       - csharp-test-runner-base
 
+  # this is only used to build the dependency for javascript-test-runner
+  # it will fail quickly and die when launched as part of the full stack
+  javascript-test-runner-base:
+    image: exercism/javascript-test-runner-base
+    build:
+      context: ../javascript-test-runner
+      dockerfile: Dockerfile
+
+  javascript-test-runner:
+    build:
+      context: ../javascript-test-runner
+      dockerfile: development_webserver.Dockerfile
+    depends_on:
+      - javascript-test-runner-base
+
```

Comment:
I'd like to make this easier to get started fro people... should I leave this commented?  Or pull it out into documentation inside README or?